qutebrowser/utils/log.py

```diff
@@ -526,6 +526,30 @@ def hide_qt_warning(pattern: str, logger: str = 'qt') -> typing.Iterator[None]:
         logger_obj.removeFilter(log_filter)
 
 
+def init_from_config(conf: typing.Any) -> None:
+    """Initialize logging settings from the config.
+
+    init_log is called before the config module is initialized, so config-based
+    initialization cannot be performed there.
+
+    Args:
+        conf: The global ConfigContainer.
+              This is passed rather than accessed via the module to avoid a
+              cyclic import.
+    """
+    ramlevel = conf.logging.level.ram
+    consolelevel = conf.logging.level.console
+    if ramlevel and ram_handler:
```

Comment:
`ram_handler`/`console_handler` are obviously set here, but `mypy` wanted me to prove they aren't `None`.

Reply:
You can add something like `assert ram_handler is not None` to make it shut up, and also express in the code that you know it's always non-None.

See https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type